docs/docs/adding-search-with-algolia.md

```diff
@@ -98,7 +105,7 @@ const postQuery = `{
         frontmatter {
           title
           slug
-          date(formatString: "MMM DD, YYYY")
+          date(formatString: "MMM D, YYYY")
```

Comment:
Why this change?

Reply:
Looks better in my opinion. Just a small oversight during the original writing of this guide but we can also keep it as is.

Reply:
Okay 👍